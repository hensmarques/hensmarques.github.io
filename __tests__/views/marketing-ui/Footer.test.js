import React from 'react';
import Footer from '../../../src/views/marketing-ui/Footer';

describe('<Footer/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
