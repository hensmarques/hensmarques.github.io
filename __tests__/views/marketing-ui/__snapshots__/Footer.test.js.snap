// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> renders and matches the snapshot 1`] = `
<div
  id="footer"
>
  <div
    className="footer-content-wrapper"
  >
    <div
      className="footer-logo-container"
    >
      <img
        className="footer-logo-img"
        src="images/logos/company-logo-white.png"
      />
      <p
        className="client-registration-id"
      >
        IBC ID Number
      </p>
    </div>
    <div
      className="footer-info-container"
    >
      <div>
        <h5>
          Address
        </h5>
        <p
          className="footer-address"
        >
          295 Madison Ave, 30th Floor,
        </p>
        <p
          className="footer-address-last"
        >
          New York, NY, 10017
        </p>
      </div>
      <div
        className="footer-item"
      >
        <h5>
          Contact
        </h5>
        <ul
          className="footer-list"
        >
          <li
            className="footer-list-item"
          >
            <a
              href="#"
            >
              Twitter
            </a>
          </li>
          <li
            className="footer-list-item"
          >
            <a
              href="#"
            >
              Facebook
            </a>
          </li>
        </ul>
      </div>
      <div
        className="footer-item"
      >
        <h5>
          Sitemap
        </h5>
        <SiteMapNav />
      </div>
    </div>
  </div>
</div>
`;
